### DISCUSSION TITLE
Parameter identification and optimization of unknown parameters

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1295] [Questions & Problems]
Hello everyone, when I read the information on the forum, I found that the Kcat value of some drug transporters or metabolic enzymes is unknown, and there is no literature or experimental source, which is obtained through parameter identification and optimization. I would like to ask you about this unknown parameter, how to set the initial value and range of parameter identification? The first picture is I find the content of the screenshots in BBS, say about unknown parameters is obtained by optimization and parameter identification, the second picture is a model screenshots, also said the unknown about Kcat, I don't have the retrieved literature value, but I don't know modeling is how to set the initial value and range, what's the basis? I have been puzzled by this question recently. I am looking forward to your answers.
![未知参数识别和优化](https://user-images.githubusercontent.com/95362728/187825532-c22310d0-229d-44de-a09e-eef6b8797166.jpg)
![参数优化例子](https://user-images.githubusercontent.com/95362728/187825554-1259baaf-9167-4f3f-8065-c3f875626426.jpg)

**Comment by wilbertdew** [comment]
Hi @WTY-tt,

Depending on the impact of the parameter, the data that you have, and the number of parameters that you are fitting, the initial parameter value and its range are more or less important. In the ideal case where you are fitting only 1 sensitive parameter with a dataset that allows its identification, you can pick a random start value and a wide range and 'Scaling' on 'Log' if you have no prior info on the possible values. It is always a good idea to first do some simulations with various values to give you a better idea of the impact of the parameter. This is also the best way to get to a proper initial value. The more parameters you are fitting simultaneously, the more data you need, and the higher the likelihood of unsatisfactory results. In the screenshot above, you have already a lot of parameters open, which might give problems or at least very long runtimes.

Best,

Wilbert

**Reply by WTY-tt** [reply]
Thank you very much for your answer. Since the obtained Vmax value of transporter has no corresponding unit in PK-SIM, I just tried to use Vmax to calculate Kcat value as the initial value to try to identify and fit parameters, but there was no way to obtain the range.I wonder if this method will work?
![p-gp Kcat计算](https://user-images.githubusercontent.com/95362728/187868658-86b80a63-5b94-4ad8-ae46-600a700fd182.jpg)
![kcat计算2](https://user-images.githubusercontent.com/95362728/187868684-61db3a4e-e927-4f19-928b-bd80f8082e5a.jpg)

**Reply by wilbertdew** [reply]
Can you specify?

**Reply by WTY-tt** [reply]
The Vmax unit of rivaroxaban P-GP obtained from the literature is pmol/min per million cells, which is not available in PK-SIM, so I don't know how to convert the units to be used in PK-SIM.

**Reply by noxavier** [reply]
Hi @wilbertdew 
If I choose only one parameter for optimization first, the optimization result will certainly be bad when other parameters are uncertain. How can I determine whether the parameter optimized first is better？
Thanks.

**Comment by wilbertdew** [comment]
@WTY-tt could you use [some assumption](https://www.tandfonline.com/doi/pdf/10.4161/cc.8.3.7608?needAccess=true) on the number of cells/mL to obtain pmol/min? 
@noxavier When faced with too many parameters to identify each realiably, one can choose to optimise them all simultaneously and accept that their values as such are only one possible combination that result in a satisfactory fit, while the value of each separate parameter remains uncertain. Either prior information or additional data is needed to reduce the uncertainty. A sensitivity analysis can help to eliminate parameters from the identification if they have no impact on the simulation result. Their uncertainty remains, but it does not necessarily translate into uncertainty of the simulations. One has to be careful here that the sensitivity analysis is done in such a way that it covers the scenarios for which simulations are required.

**Reply by WTY-tt** [reply]
Hello, thank you very much for your answer. I have not found an intermediate unit that can be converted.
